web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -184,6 +187,30 @@ class SpeakPage extends React.Component<Props, State> {
     return this.state.recordingStatus === 'recording';
   }
 
+  private keyUprerecording = async (event: any) => {
+    let index = null;
+    //for both sets of number keys on a keyboard with shift key
+    if (event.which === 35 || event.which === 49) {
```

Comment:
Can we use `event.key` instead? It's more descriptive than numbers.

Reply:
yes, but then there will be little change in value returned from event
In case of event.key ->      shift +1 = "end" or "!"
                                          shift +2 = "arrowdown " or "@"
                                         or more

Reply:
Good point! `event.code` should do the trick! That would yield `Digit1` (even when shift is pressed).

Reply:
is there something left to be done in this PR.

Reply:
If you could switch to using `event.code` that'd be great. That way it'd be clear which keys do what, just by reading the code.

Reply:
awesome. Done